ui/src/views/Imapsync.vue

```diff
@@ -0,0 +1,861 @@
+<!--
+#
+# Copyright (C) 2019 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+-->
+
+<template>
+  <div>
+    <h2>{{ $t('imapsync.title') }}</h2>
+    <doc-info
+      :placement="'top'"
+      :title="$t('docs.imapsync')"
+      :chapter="'mail'"
+      :section="'mailbox-configuration'"
```

Comment:
This should point to the correct anchor which should be something like `email-imap-synchronization`

Please, build the manual and search for correct anchor name

Reply:
done